language/synth.metadata

```diff
@@ -1,48 +1,11 @@
 {
-  "updateTime": "2019-04-04T12:17:35.761105Z",
```

Comment:
Is there a need to keep the changes in the language directory?

Reply:
+1. It seems this was an extra change in addition to the translate work

Reply:
Thank you!